src/TextArea/TextArea.less

```diff
@@ -78,10 +78,6 @@
   .border--s(@neutral_silver);
   color: @neutral_dark_gray;
   background: @neutral_silver;
-
-  .TextArea--input {
```

Comment:
Unneeded because the `TextArea` input is already [transparent](https://github.com/Clever/components/pull/123/files#diff-f33c3b7a08d6659c4f1c4293d40f34cbR47)